feat(messaging): add FAQ (#1156)
scaleway/docs-content: faq/messaging-and-queuing.mdx

## What is NATS / SNS / SQS ?

NATS, SNS and SQS are all messaging protocols supported by Scaleway Messaging and Queuing. You can find out more about these protocols, and other essential Messaging and Queuing concepts, on our dedicated [concepts page](/compute/messaging/concepts/).

## Can I subscribe to a topic that is in a different namespace?

It is currently not possible for an SQS queue to subscribe to a topic located in a different namespace.

## Is the Scaleway Messaging and Queuing SQS gateway compatible with my application/framework/tool?

We currently implement the API endpoints listed [here](/compute/messaging/reference-content/sqs-support/), which makes Scaleway Messaging and Queuing compatible with the AWS SDK as well as many other tools and framework including KEDA and Symfony. Note that you need to specify both Regions and URL to ensure compatibility.

## Does Scaleway Messaging and Queuing support all SNS features?

The following subscriber types are supported:
- HTTP/S clients

> - Serverless functions

Author: ```suggestion
- Serverless Functions
```

- SQS queues in the same namespace

> For a full details of supported and unsupported SNS features, see our [dedicated page](/compute/messaging/reference-content/sns-support/)

Reviewer: ```suggestion
For more details on supported and unsupported SNS features, see our [dedicated page](/compute/messaging/reference-content/sns-support/)
```

> For a full details of supported and unsupported SNS features, see our [dedicated page](/compute/messaging/reference-content/sns-support/)

Author: ```suggestion
For more details on supported and unsupported SNS features, see our [dedicated page](/compute/messaging/reference-content/sns-support/)
```
